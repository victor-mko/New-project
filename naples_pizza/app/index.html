<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>MyProject</title>

    <link href="https://fonts.googleapis.com/css?family=Pangolin&amp;subset=cyrillic,cyrillic-ext" rel="stylesheet">

    
	<link rel="stylesheet" href="css/libs.css">
	<link rel="stylesheet" href="css/main.css">
	
</head>
<body>


    <script type="text/javascript" src="js/libs.min.js"></script>
    <script type="text/javascript" src="js/common.js"></script>
</body>
</html>