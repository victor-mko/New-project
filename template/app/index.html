<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>MyProject</title>

	<link rel="stylesheet" href="css/libs.css">
	<link rel="stylesheet" href="css/main.css">
	
</head>
<body>
	<div class="tariffs">
        <div class="tariff">
            <h3>1 month Membership</h3>
            <p>Rebils at $7.95 per month until cancelled*</p>
            <span>$9.95</span>
        </div>
        <div class="tariff">
            <h3>1 month Membership</h3>
            <p>Rebils at $7.95 per month until cancelled*</p>
            <span>$9.95</span>
        </div>
        <div class="tariff">
            <h3>1 month Membership</h3>
            <p>Rebils at $7.95 per month until cancelled*</p>
            <span>$9.95</span>
        </div>
    </div>

    <script type="text/javascript" src="js/libs.min.js"></script>
    <script type="text/javascript" src="js/common.js"></script>
</body>
</html>