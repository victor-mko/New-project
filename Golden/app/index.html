<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Golden</title>

    <link rel="stylesheet" href="css/libs.css">
    <link rel="stylesheet" href="css/main.css">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>

</head>

    
<body>
    <div id="header-box" class="header-box">
        <header class="header container" id="header">
            <div class="row nav-logo-box">
                <div class="logo WisdomScriptAJ left">
                    <a href="#services" title="Golden">Golden</a>
                </div>
                <nav class="right" id="menu">
                    <ul>
                        <li class="text-left left" id="nav-services">
                            <a href="#services">SERVICES</a>
                        </li>
                        <li class="text-left left" id="nav-portfolio">
                            <a href="#portfolio">PORTFOLIO</a>
                        </li>
                        <li class="text-left left" id="nav-about">
                            <a href="#about">ABOUT</a>
                        </li>
                        <li class="text-left left" id="nav-teem">
                            <a href="#teem">TEEM</a>
                        </li>
                        <li class="text-left left" id="nav-contact">
                            <a href="#contact">CONTACT</a>
                        </li>
                    </ul>
                </nav>
            </div>
        </header>
    </div>
    <div class="home-box" id="header">
        <div class="overlay">
            <div class="home container">
                <div class="home-slogan row">
                    <h2 class="text-center">Welcome To Our Studio!</h2>
                    <h1 class="text-center">Itâ€™s nice to Meet you</h1>
                    <a href="#services" class="btn">Tell Me More</a>
                </div>
            </div>
        </div>
    </div>
    <section id="services" class="container">
        <div class="section-title row text-center">
            <h2>Services</h2>
            <p>Proin iaculis purus consequat sem cure.</p>
        </div>
        <div class="row servies-box text-center">
            <div class="col-sm-12 col-md-4">
                <div class="icon-servies-box center">
                    <img src="img/icon-e-commerce.png" alt=""></div>
                <h3>E-Commerce</h3>
                <p>
                    Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.
                </p>
            </div>
            <div class="col-sm-12 col-md-4">
                <div class="icon-servies-box center">
                    <img src="img/icon-responsive.png" alt=""></div>
                <h3>Responsive Web</h3>
                <p>
                    Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.
                </p>
            </div>
            <div class="col-sm-12 col-md-4">
                <div class="icon-servies-box center">
                    <img src="img/icon-lockes.png" alt=""></div>
                <h3>Web Security</h3>
                <p>
                    Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.
                </p>
            </div>
        </div>
    </section>
    <section id="portfolio" class="portfolio">
        <div class="portfolio-box container">
            <div class="section-title row text-center">
                <h2>OUR PORTFOLIO</h2>
                <p>Proin iaculis purus consequat sem cure.</p>
            </div>
            <div class="row servies-box text-center">
                <div class="portfolio-item col-xs-12 col-sm-6 col-md-4">
                    <a href="" class="portfolio-link">
                        <image src="img/bg-portfolio-1.png"></image>
                        <div class="portfolio-image-hover">+</div>
                    </a>
                    <div class="portfolio-caption">
                        <h3>Ebony & Ivory</h3>
                        <p>Branding</p>
                    </div>
                </div>
                <div class="portfolio-item col-xs-12 col-sm-6 col-md-4">
                    <a href="" class="portfolio-link">
                        <image src="img/bg-portfolio-2.png"></image>
                        <div class="portfolio-image-hover">+</div>
                    </a>
                    <div class="portfolio-caption">
                        <h3>Smart Stationary</h3>
                        <p>Print Design</p>
                    </div>
                </div>
                <div class="portfolio-item col-xs-12 col-sm-6 col-md-4">
                    <a href="" class="portfolio-link">
                        <image src="img/bg-portfolio-3.png"></image>
                        <div class="portfolio-image-hover">+</div>
                    </a>
                    <div class="portfolio-caption">
                        <h3>Clever Poster</h3>
                        <p>Print Design</p>
                    </div>
                </div>
                <div class="portfolio-item col-xs-12 col-sm-6 col-md-4">
                    <a href="" class="portfolio-link">
                        <image src="img/bg-portfolio-4.png"></image>
                        <div class="portfolio-image-hover">+</div>
                    </a>
                    <div class="portfolio-caption">
                        <h3>Vinyl Record</h3>
                        <p>Product Mock-Up</p>
                    </div>
                </div>
                <div class="portfolio-item col-xs-12 col-sm-6 col-md-4">
                    <a href="" class="portfolio-link">
                        <image src="img/bg-portfolio-5.png"></image>
                        <div class="portfolio-image-hover">+</div>
                    </a>
                    <div class="portfolio-caption">
                        <h3>Treehouse Template</h3>
                        <p>Web Design</p>
                    </div>
                </div>
                <div class="portfolio-item col-xs-12 col-sm-6 col-md-4">
                    <a href="" class="portfolio-link">
                        <image src="img/bg-portfolio-6.png"></image>
                        <div class="portfolio-image-hover">+</div>
                    </a>
                    <div class="portfolio-caption">
                        <h3>Burned Logo</h3>
                        <p>Branding</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section id="about" class="about">
        <div class="about-box container">
            <div class="section-title row text-center">
                <h2>ABOUT US</h2>
                <p>Proin iaculis purus consequat sem cure.</p>
            </div>
            <div class="row chronology-box">
                <div class="col-xs-4 text-right left-content">
                    <h3 class="date">JULY 2010</h3>
                    <h3 class="event">Our Humble Beginnings</h3>
                    <p class="event-text">
                        Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.
                    </p>
                </div>
                <div class="col-xs-4">
                    <div class="event-circle event-1"></div>
                </div>
            </div>
            <div class="vertical-line"></div>
            <div class="row chronology-box">
                <div class="col-xs-4 col-xs-offset-4">
                    <div class="event-circle event-2"></div>
                </div>
                <div class="col-xs-4 text-left right-content">
                    <h3 class="date">JANUARY 2011</h3>
                    <h3 class="event">Facing Startup Battles</h3>
                    <p class="event-text">
                        Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.
                    </p>
                </div>
            </div>
            <div class="vertical-line"></div>
            <div class="row chronology-box">
                <div class="col-xs-4 text-right left-content">
                    <h3 class="date">DECEMBER 2012</h3>
                    <h3 class="event">Enter The Dark Days</h3>
                    <p class="event-text">
                        Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.
                    </p>
                </div>
                <div class="col-xs-4">
                    <div class="event-3 event-circle"></div>
                </div>
            </div>
            <div class="vertical-line"></div>
            <div class="row chronology-box">
                <div class="col-xs-4 col-xs-offset-4">
                    <div class="event-circle event-4"></div>
                </div>
                <div class="col-xs-4 text-left right-content">
                    <h3 class="date">FEBRUARY 2014</h3>
                    <h3 class="event">Our Triumph</h3>
                    <p class="event-text">
                        Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.
                    </p>
                </div>
            </div>
            <div class="vertical-line"></div>
            <div class="row chronology-box">
                <div class="col-xs-4 col-xs-offset-4">
                    <div class="event-circle continue-story">
                        <p>
                            OUR
                            <br>
                            STORY
                            <br>
                            CONTINUE
                            <br>...</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section id="teem" class="teem">
        <div class="container teem-box">
            <div class="section-title row text-center">
                <h2>OUR AMAZING TEAM</h2>
                <p>Proin iaculis purus consequat sem cure.</p>
            </div>
            <div class="row teem-member-box">
                <div class="col-md-4">
                    <div class="teem-member">
                        <div class="teem-member-photo"></div> <strong>Kimberly Thompson</strong>
                        <p>Marketer</p>
                        <div class="soc-icon-box">
                            <a class="twitter-icon" href="#"> <i class="icon-twitter"></i>
                            </a>
                            <a class="facebook-f-icon" href="#"> <i class="icon-facebook-f"></i>
                            </a>
                            <a class="linkedin-icon" href="#">
                                <i class="icon-linkedin"></i>
                            </a>
                        </div>
                    </div>
                </div>
                <div class="col-md-4">
                    <div class="teem-member">
                        <div class="teem-member-photo"></div> <strong>Rico Massimo</strong>
                        <p>Coder</p>
                        <div class="soc-icon-box">
                            <a class="twitter-icon" href="#">
                                <i class="icon-twitter"></i>
                            </a>
                            <a class="facebook-f-icon" href="#">
                                <i class="icon-facebook-f"></i>
                            </a>
                            <a class="linkedin-icon" href="#">
                                <i class="icon-linkedin"></i>
                            </a>
                        </div>
                    </div>
                </div>
                <div class="col-md-4">
                    <div class="teem-member">
                        <div class="teem-member-photo"></div>
                        <strong>Uku Mason</strong>
                        <p>Graphic Designer</p>
                        <div class="soc-icon-box">
                            <a class="facebook-f-icon" href="#">
                                <i class="icon-facebook-f"></i>
                            </a>
                            <a class="linkedin-icon" href="#">
                                <i class="icon-linkedin"></i>
                            </a>
                        </div>
                    </div>
                </div>
            </div>
            <div class="about-teem">
                Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt proin iaculis.
            </div>
        </div>
    </section>
    <section id="clients" class="clients">
        <div class="container">
            <div class="row">
                <div class="col-md-3">
                    <a href="#">
                        <img src="img/envato-img.png" alt=""></a>
                </div>
                <div class="col-md-3">
                    <a href="#">
                        <img src="img/WP-img.png" alt=""></a>
                </div>
                <div class="col-md-3">
                    <a href="#">
                        <img src="img/tuts-img.png" alt=""></a>
                </div>
                <div class="col-md-3">
                    <a href="#">
                        <img src="img/microlanser-img.png" alt=""></a>
                </div>
            </div>
        </div>
    </section>
    <section id="contact" class="contact">
        <div class="container">
            <div class="section-title row text-center">
                <h2>CONTACT US</h2>
                <p>Proin iaculis purus consequat sem cure.</p>
            </div>
            <form class="row">
                <div class="col-md-6">
                    <input type="text" name="name" placeholder="YOUR NAME*" required="required">
                    <input type="email" name="email" required="required" placeholder="YOUR EMAIL*">
                    <input type="tel" name="phone" required="required" placeholder="YOUR PHONE*">
                </div>
                <div class="col-md-6">
                    <textarea class="message" name="message" placeholder="YOUR MESSAGE"></textarea>
                </div>
                <button class="btn">SEND MESSAGE</button>
            </form>
        </div>
    </section>
    <footer>
        <div class="container">
            <div class="row">
                <div class="col-md-4 footer-text text-left">Â© Copyright 2014 FreebiesXpress.com</div>
                <div class="col-md-4 text-center">
                    <div class="soc-icon-box">
                        <a class="twitter-icon" href="#">
                            <i class="icon-twitter"></i>
                        </a>
                        <a class="linkedin-icon" href="#">
                            <i class="icon-linkedin"></i>
                        </a>
                        <a class="facebook-f-icon" href="#">
                            <i class="icon-facebook-f"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    </footer>
    <script type="text/javascript" src="js/libs.min.js"></script>
    <script type="text/javascript" src="js/common.js"></script>
</body>
</html>